#!/usr/bin/env python
PACKAGE = "mbzirc_husky"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
gen.add("tolerance", double_t, 0, "Ransac tolerance",   0.125 , 0,  3)
gen.add("angleTolerance", double_t, 0, "Angle Ransac tolerance",   0.5 , 0,  4)
gen.add("distanceTolerance", double_t, 0, "Distance tolerance",   0.25 , 0,  2)
gen.add("distance", double_t, 0, "Distance",   2.8 , 0,  10)
gen.add("fwSpeed", double_t, 0, "Forward Speed",   0.1 , 0,  10)
gen.add("minPoints", int_t, 0, "Minimum points",   0 , 0,  100)
gen.add("maxEval", int_t, 0, "Maximal evaluation Points",   0 , 0,  1000)
gen.add("spdLimit", double_t, 0, "Speed Limit",   0.3 , 0,  10)
gen.add("cameraXOffset", double_t, 0, "cameraXOffset",   -0.02 , -0.2,  0.2)
gen.add("cameraYOffset", double_t, 0, "cameraYOffset",   -0.02 , -0.2,  0.2)
gen.add("cameraXAngleOffset", double_t, 0, "cameraXAngleOffset",   0 , -0.2,  0.2)
gen.add("cameraYAngleOffset", double_t, 0, "cameraYAngleOffset",   0 , -0.2,  0.2)


exit(gen.generate(PACKAGE, "detectBrick", "detectBrick"))

